face-analyzer/src/views/stimuli/index.js

```diff
@@ -60,6 +60,7 @@ const Stimuli = () => {
     };
 
     const saveReaction = () => {
+        // eslint-disable-next-line no-unused-vars
         saveNewReaction(stimuliId).then(result => {
```

Comment:
I'm pretty sure I fixed this in every single one of my branches, I don't understand how it's still here 😄